entry.sh

```diff
@@ -6,6 +6,8 @@ set -o errexit
     mkdir -p /dev/net
 [ -c /dev/net/tun ] ||
     mknod /dev/net/tun c 10 200
+[ -d /mnt/root/tmp/resin-supervisor ] ||
+    mkdir -p /mnt/root/tmp/resin-supervisor
```

Comment:
Not necessarily related to this PR, but if you use the -p flag with mkdir it should be safe to run it whether the directory exists or not.